R/env.R

```diff
@@ -0,0 +1,702 @@
+#' Get an environment.
+#'
+#' Environments are objects that create a scope for evaluation of R
+#' code. Reification of scope is one of the most powerful feature of
```

Comment:
I think so. The idea is that environments are scopes reified as first-class objects.